relaton/serializers/bibxml/reference.py

```diff
@@ -89,7 +89,12 @@ def create_reference(item: BibliographicItem) -> Element:
     if len(abstracts) > 0:
         front.append(create_abstract(abstracts))
 
-    ref = E.reference(front)
+    # refcontent
+    refcontent = create_refcontent(item.extent)
+    if refcontent:
+        ref = E.reference(front, E.refcontent(refcontent))
+    else:
+        ref = E.reference(front)
```

Comment:
We already use `ref` as mutable element to which we append as we go.

Instead of this, the better approach is to leave the initial `ref` assignment as is, and *if* there is refcontent then append that element to `ref`.

Reply:
Did it here: https://github.com/relaton/relaton-py/commit/c9121673457280f69dda29ed4943188c964d8430#diff-fc6c843ab71694a81afbf85fd6f7fdeebbdacbc6a219803f7eba800c900d3ff8R92-R95